IdentityCore/src/configuration/webview/MSIDBaseWebRequestConfiguration.h

```diff
@@ -73,6 +77,15 @@ NS_ASSUME_NONNULL_BEGIN
               ignoreInvalidState:(BOOL)ignoreInvalidState
                       ssoContext:(nullable MSIDExternalSSOContext *)ssoContext;
 
+#if MSAL_JS_AUTOMATION
+- (instancetype)initWithStartURL:(NSURL *)startURL
+                  endRedirectUri:(NSString *)endRedirectUri
+                           state:(NSString *)state
+              ignoreInvalidState:(BOOL)ignoreInvalidState
+                      javascript:(nullable NSString *)javascript
+                      ssoContext:(nullable MSIDExternalSSOContext *)ssoContext;
+#endif
+
```

Comment:
Maybe considering subclass as `MSIDBaseWebRequestConfigurationMock`

Reply:
After syncing with Kai offline, I feel it's not necessary to make this change. The decision is still keeping the current logic.